go.mod

```diff
@@ -69,6 +69,7 @@ require (
 )
 
 replace (
+	github.com/containers/image/v4 => github.com/containers/image/v4 v4.0.0-20191004175620-9da78d1f50d2
```

Comment:
note that this may not be necessary, but was removed in original PR

Reply:
confirmed commit 9da78d1f50d2 was the level for my containers/image fix for handling mounted secrets container `.dockercfg` files